canonical-web-and-design/www.ubuntu.com
# Refresh /esm for ua-i

## Done

- Updated the /esm page per the copy doc

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/esm](http://0.0.0.0:8001/esm)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1BK-XXECYJJllG8jIe3WN3X8EMo_3wBBa03nlzYDP8xE/edit#heading=h.1hwrj8tslacn)

## Issue / Card

Fixes #5016
